src/redux/actions/samples/createSample.js

```diff
@@ -66,17 +62,17 @@ const createSample = (
 
     throwIfRequestFailed(response, json, endUserMessages.ERROR_SAVING);
 
-    dispatch({
+    await dispatch({
       type: SAMPLES_CREATE,
       payload: { sample: newSample, experimentId },
     });
 
-    dispatch(saveExperiment(experimentId));
```

Comment:
where did `saveExperiment` go?

Reply:
I think we don't need it anymore because now the sample creation endpoint is smarter and also updates the experiment in dynamo.

Reply:
makes sense, thanks for clarifying